command/agent.go

```diff
@@ -1047,3 +1050,19 @@ func (c *AgentCommand) handleMetrics() http.Handler {
 		}
 	})
 }
+
+func (c *AgentCommand) handleQuit() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// Only handle POST/PUT requests
+		switch r.Method {
+		case http.MethodPost:
+		case http.MethodPut:
```

Comment:
I think `case` can take more than one argument.
```suggestion
		case http.MethodPost, http.MethodPut:
```